ParquetClassLibrary/Characters/BeingStatus.cs

```diff
@@ -1,37 +1,130 @@
+using System.Collections.Generic;
 using Newtonsoft.Json;
 using ParquetClassLibrary.Sandbox;
 
 namespace ParquetClassLibrary.Characters
 {
     public class BeingStatus
     {
+        /// <summary>
+        /// Describes the version of serialized data.
+        /// Allows selecting data files that can be successfully deserialized.
+        /// </summary>
+        public readonly string DataVersion = AssemblyInfo.SupportedBeingDataVersion;
+
+        /// <summary>Tracks how many times the data structure has been serialized.</summary>
+        public int Revision { get; private set; }
+
         /// <summary>The <see cref="Being"/> whose status is being tracked.</summary>
         [JsonProperty(PropertyName = "in_beingDefinition")]
         public Being BeingDefinition { get; }
 
         /// <summary>The <see cref="Location"/> the tracked <see cref="Being"/> occupies.</summary>
-        [JsonProperty(PropertyName = "in_currentLocation")]
-        public Location CurrentLocation { get; set; }
+        [JsonProperty(PropertyName = "in_position")]
+        public Location Position { get; set; }
+
+        /// <summary>The <see cref="Location"/> the tracked <see cref="Being"/> will next spawn at.</summary>
+        /// <remarks>For example, for <see cref="PlayerCharacter"/>s this is their last save spot.</remarks>
+        [JsonProperty(PropertyName = "in_spawnAt")]
+        public Location SpawnAt { get; set; }
 
         /// <summary>The <see cref="Behavior"/> currently governing the tracked <see cref="Being"/>.</summary>
         [JsonProperty(PropertyName = "in_currentBehavior")]
         public Behavior CurrentBehavior { get; set; }
 
-        // IDEA I expect more will be added here as design goes on.
+        /// <summary>The time remaining that the tracked <see cref="Being"/> can safely remain in the current <see cref="Sandbox.IDs.Biome"/>.</summary>
+        /// <remarks>It is likely that this will only be used by <see cref="PlayerCharacter"/>.</remarks>
+        [JsonProperty(PropertyName = "in_biomeTimeRemaining")]
+        public int BiomeTimeRemaining { get; set; }
+
+        /// <summary>The time it takes the tracked <see cref="Being"/> to place new parquets.</summary>
+        [JsonProperty(PropertyName = "in_buildingSpeed")]
+        public float BuildingSpeed { get; set; }
```

Comment:
Do you imagine that characters other than the player character will be building? A few placeholder stats are interesting, but there's a good chance these will become universal constants later, or something that the tool modifies.

Reply:
Mmmm, that is a good point.
I think, though, that since it is still early days design-wise I'm going to leave them modifiable for now.  I feel like it will be easier to go from modifiable to constant than vice versa.